### DISCUSSION TITLE
Release中OldGPU的含义

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [xfangfang/wiliwili#414] [Q&A] [answered]
v1.5.0的release中Windows平台增加了带有OldGPU后缀的文件，这些与不带后缀的文件有什么区别呢？哪些GPU算OldGPU呢？能否在Readme或release中说明一下呢？

**Comment by xfangfang** [comment, marked as answer]
不带后缀版本使用d3d11图形api，带后缀版本使用OpenGL3.2+

如果能正常运行默认版，就不要用OldGPU版，默认版性能更好。

正常来说应该按照具体使用的图形api来标注，但是某些包管理器会查询release中的文件名来自动更新，为了避免增加维护包管理器的朋友的工作量，将OpenGl版本标注为了OldGPU。

一般来说十年内的电脑都能正常运行默认版。

**Reply by FrzMtrsprt** [reply]
好的，感谢
